backend/drive/drive.go

```diff
@@ -299,6 +299,11 @@ a non root folder as its starting point.
 			Default:  true,
 			Help:     "Send files to the trash instead of deleting permanently.\n\nDefaults to true, namely sending files to the trash.\nUse `--drive-use-trash=false` to delete files permanently instead.",
 			Advanced: true,
+		}, {
+			Name:     "copy_shortcut_content",
+			Default:  false,
+			Help:     "Copy the files instead of just shortcuts when performing a server-side transfer.\nUse `--drive-copy-shortcut-content=true` to enable it. ",
```

Comment:
The first line of this should be what you see when you run `rclone help flags` or `rclone help flags copy-shortcut` so it shouldn't be too long.

There should then be a `\n\n` and the help to go in the help text.

Its easier to write the help if you use multiple lines - see "root_folder_id" for an example.

Note also that `--drive-copy-shortcut-content=true` is equivalent to `--drive-copy-shortcut-content` and you probably don't need to mention it.

So I'd suggest something like

```go
Help: `Server side copy contents of shortcuts instead of the shortcut.

When doing server side copies, normally rclone will copy shortcuts as
shortcuts.

If this flag is used then rclone will copy the contents of shortcuts
rather than shortcuts themselves when doing server side copies.
```